--- exercises/026_hello2.zig	2025-12-27 21:15:21.403899392 +0100
+++ answers/026_hello2.zig	2025-12-27 20:28:53.267236800 +0100
@@ -27,5 +27,5 @@
     // to be able to pass it up as a return value of main().
     //
     // We just learned of a single statement which can accomplish this.
-    stdout.interface.print("Hello world!\n", .{});
+    try stdout.interface.print("Hello world!\n", .{});
 }
