--- exercises/028_defer2.zig	2026-06-02 06:08:12.713672612 +0200
+++ answers/028_defer2.zig	2026-06-02 06:08:43.262234023 +0200
@@ -20,7 +20,7 @@
 fn printAnimal(animal: u8) void {
     std.debug.print("(", .{});
 
-    std.debug.print(") ", .{}); // <---- how?!
+    defer std.debug.print(") ", .{}); // <---- how?!
 
     if (animal == 'g') {
         std.debug.print("Goat", .{});
@@ -51,9 +51,9 @@
 
     // Try reordering the statements to get the answer 42
     {
-        defer x = x / 10;
-        defer x = x + 11;
         defer x = x * 2;
+        defer x = x + 11;
+        defer x = x / 10;
     }
 
     return x;
