--- exercises/034_quiz4.zig	2025-12-28 14:43:41.087943476 +0100
+++ answers/034_quiz4.zig	2025-12-28 14:42:23.878472164 +0100
@@ -10,11 +10,11 @@
 
 const NumError = error{IllegalNumber};
 
-pub fn main() void {
+pub fn main() !void {
     var stdout_writer = std.Io.File.stdout().writer(io, &.{});
     const stdout = &stdout_writer.interface;
 
-    const my_num: u32 = getNumber();
+    const my_num: u32 = try getNumber();
 
     try stdout.print("my_num={}\n", .{my_num});
 }
