--- exercises/060_floats.zig	2026-05-02 19:22:46.225370223 +0200
+++ answers/060_floats.zig	2026-05-02 19:22:47.523142218 +0200
@@ -50,7 +50,7 @@
     //
     // We'll convert this weight from pounds to metric units at a
     // conversion of 0.453592 kg to the pound.
-    const shuttle_weight: f16 = 0.453592 * 4480e3;
+    const shuttle_weight: f32 = 0.453592 * 4480e3;
 
     // By default, float values are formatted in standard decimal
     // notation. Experiment with '{d}' and '{d:.3}' to see how
@@ -58,7 +58,7 @@
     // scientific notation.
     // NOTE: The weight of the shuttle is a huge number, a scientific notation
     // may be more appropriate.
-    print("Shuttle liftoff weight: {d:.0} metric tons\n", .{shuttle_weight / 1e3});
+    print("Shuttle liftoff weight: {e:.3} metric tons\n", .{shuttle_weight / 1e3});
 }
 
 // Floating further:
