--- exercises/074_comptime9.zig	2026-04-30 19:28:08.990995153 +0200
+++ answers/074_comptime9.zig	2026-04-30 19:27:09.642728692 +0200
@@ -28,12 +28,12 @@
         start, // Ready to start a new animal.
         l, // This means we've seen an "l", so if we see an "m", we know it's a Llama.
     };
-    var state = State.start;
+    comptime var state = State.start;
 
     // We return an array of animals representing the creature. (This is why we
     // really needed the 'count' parameter. Arrays need a size.)
     var animals: [count]Animal = .{undefined} ** count;
-    var next_animal: usize = 0;
+    comptime var next_animal: usize = 0;
 
     inline for (fmt) |char| {
 
@@ -57,7 +57,7 @@
                 //
                 // What do you think happens with Gators? Do they join with
                 // other animals or is this an error?
-                'g' => ???,
+                'g' => @compileError("Gators refuse to join with other animals."),
 
                 else => @compileError(std.fmt.comptimePrint("No animal starts with '{c}'!", .{char})),
             },
@@ -69,7 +69,7 @@
                     next_animal += 1;
                     // Something is missing here. After we finish a Llama, we
                     // need to be ready to _start_ over with a new animal...
-                    ???
+                    state = .start;
                 },
 
                 else => @compileError("Only llamas start with 'l'!"),
