--- exercises/090_async7.zig	2026-04-02 10:36:42.910708919 +0200
+++ answers/090_async7.zig	2026-04-02 10:36:51.965884223 +0200
@@ -49,8 +49,8 @@
     for (0..times) |_| {
         // Acquire the lock before modifying shared state.
         // What Mutex method blocks until the lock is acquired?
-        state.mutex.??? catch return;
-        defer state.mutex.unlock(); // <-- what's missing here?
+        state.mutex.lock(io) catch return;
+        defer state.mutex.unlock(io);
 
         state.counter += 1;
     }
