--- exercises/093_async9.zig	2026-04-13 17:55:35.567204530 +0200
+++ answers/093_async9.zig	2026-04-13 18:05:05.636355044 +0200
@@ -43,7 +43,7 @@
     // Launch with a guaranteed separate unit of concurrency.
     // Which Io method guarantees this?
     // (Hint: unlike io.async, this one can fail!)
-    var future = try io.???(compute, .{io});
+    var future = try io.concurrent(compute, .{io});
     defer _ = future.cancel(io);
 
     // Note: All breaks in this exercise (using sleep)
