--- exercises/093_async9.zig	2026-04-14 22:06:23.644776893 +0200
+++ answers/093_async9.zig	2026-04-14 22:06:30.606912044 +0200
@@ -63,10 +63,10 @@
 
     // Launch two workers, each searching half the array.
     // Remember, we want them to be guaranteed separate units of concurrency.
-    var f1 = ???(searchRange, .{ data[0..mid], target, 0, 0, &queue, io });
+    var f1 = try io.concurrent(searchThreshold, .{ io, data[0..mid], threshold, 0, 0, &queue });
     defer _ = f1.cancel(io);
 
-    var f2 = ???(searchRange, .{ data[mid..], target, mid, 1, &queue, io });
+    var f2 = try io.concurrent(searchThreshold, .{ io, data[mid..], threshold, mid, 1, &queue });
     defer _ = f2.cancel(io);
 
     // Wait for the first result.
