--- exercises/112_vectors.zig	2025-08-15 15:17:57.840348083 +0200
+++ answers/112_vectors.zig	2026-04-02 10:51:15.819831812 +0200
@@ -121,8 +121,8 @@
 
 const Vec4 = @Vector(4, f32);
 fn calcMaxPairwiseDiffNew(a: Vec4, b: Vec4) f32 {
-    const abs_diff_vec = ???;
-    const max_diff = @reduce(???, abs_diff_vec);
+    const abs_diff_vec = @abs(a - b);
+    const max_diff = @reduce(.Max, abs_diff_vec);
     return max_diff;
 }
 
