--- exercises/116_defer3.zig	2026-05-31 22:29:56.189323732 +0200
+++ answers/116_defer3.zig	2026-05-31 22:30:17.749186667 +0200
@@ -7,9 +7,9 @@
     var x: u32 = 100;
     {
         // Try reordering the statements to get the answer 42
-        defer x = x / 10;
-        defer x = x + 11;
         defer x = x * 2;
+        defer x = x + 11;
+        defer x = x / 10;

         // It might seem silly in this example, but it's important to know when
         // deinitializing containers whose elements need to be deinitialized first.
